UI.tsv

```diff
@@ -705,17 +705,17 @@ UI_20150317_000704	{s20}{#050505}Describe whatever
 UI_20150317_000705	{@st41} All {/}
 UI_20150317_000706	{@st41} Party {/}
 UI_20150317_000707	{@st41} Monster {/}
-UI_20150317_000708	{@st41b}{#f0dcaa}Remaining Status Points ▶ {@st46}%s
-UI_20150317_000709	{@st59b}My Information{nl}{@st59s}- Status : Earn points by level up and use it to enhance 5 abilities. {nl}- Information : Character info based on current status and equipment {nl}- Title : Can be earned by accomplishing Achievements {nl} {/}
+UI_20150317_000708	{@st41b}{#f0dcaa}Remaining Ability Points ▶ {@st46}%s
+UI_20150317_000709	{@st59b}My Information{nl}{@st59s}- Ability : By leveling up, you can earn points that you can use to enhance the 5 available abilities to become more powerful. {nl}- Information : Character info based on current status and equipment {nl}- Title : Can be earned by accomplishing achievements {nl} {/}
 UI_20150317_000710	{@st41b}Info{/}
 UI_20150317_000711	{@st41b}Title{/}
 UI_20150317_000712	Show Class
 UI_20150317_000713	Show Character Name
 UI_20150317_000714	{@st59}Change character name{/}
 UI_20150317_000715	{@st41b}Apply
-UI_20150317_000716	{@st59}Apply status allocation{/}
```

Comment:
I don't see the point in the extra text. Apply and Cancel are just fine. Players will know fine what changes they are applying or canceling based on the window they have open.